aurora/src/main/java/com/market/banica/aurora/config/ChannelManager.java

```diff
@@ -122,6 +123,7 @@ private ManagedChannel buildChannel(String host, int port) {
                 .usePlaintext()
                 .enableRetry()
                 .defaultServiceConfig(ChannelRPCConfig.getInstance().getServiceConfig())
+                .maxRetryAttempts(MAX_RETRY_ATTEMPTS)
```

Comment:
Why have you added the max attempts here, when you have it in the configuration file?

Reply:
If we specify a greater number than 5 for "maxAttempts" in the configuration file and we don't implicitly add "maxRetryAttemps" on a channel build, gRPC decreases the "maxAttemps" back to 5 (by default).

Reply:
Ok, thanks. In this case do we need to have it specified in the config file as well?

Reply:
It is also specified in the config file 'from config' ->  '"maxAttempts": 1000'. The attempts specified in config file and on channel creations are the same.